jigasi/rootfs/etc/services.d/jigasi/run

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/with-contenv bash
+if [[ -f /config/key.json ]]; then
```

Comment:
Also check the transcriber type I think, simpler for testing switching between them.